timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -210,15 +212,15 @@ private LockService createThreadPoolingLockService(long slowLogTriggerMillis) {
         return new ThreadPooledLockService(lockServiceNotUsingThreadPooling, localThreadPoolSize, sharedThreadPool);
     }
 
-    private LockService createTimeLimitedLockService(long slowLogTriggerMillis) {
+    private CloseableRemoteLockService createTimeLimitedLockService(long slowLogTriggerMillis) {
         LockServerOptions lockServerOptions = new LockServerOptions() {
             @Override
             public long slowLogTriggerMillis() {
                 return slowLogTriggerMillis;
             }
         };
 
-        LockService rawLockService = LockServiceImpl.create(lockServerOptions);
+        LockServiceImpl rawLockService = LockServiceImpl.create(lockServerOptions);
```

Comment:
`LockServiceImpl` isn't a `CloseableRemoteLockService` though (it implements both interfaces, but separately). Since this is in lock-api it's probably OK to have it just extend CRLS instead of C and RLS separately?